src/css/components/panel.scss

```diff
@@ -68,61 +69,44 @@ h4 + .panel {
 .panel-group {
   @include clearfix;
 
-  margin-bottom: 0.65rem;
-
   & + .panel-group {
     margin-top: $grid-4;
   }
 }
 
 .panel-actions {
-  > * {
-    margin-right: 2rem;
+  margin-top: $grid-4;
 
-    &:last-child {
-      margin-right: 0;
-    }
+  .action {
```

Comment:
Is there a way we can do this without nesting actions? There may not be and that's fine, just wanted to point out.